stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -150,18 +167,43 @@ private class TraceContext {
                 outputs: [CTensorHandle]) {
     internalConsistencyCheck(traceGraphFn == nil)
     var symbolicOutputs: [TF_Output] = []
+    // Only add symbolic output tensors as the outputs of the trace graph function.
+    // For example, let the tracee be:
+    //   func foo(x: Tensor) -> (Tensor, Tensor) {
+    //     let y = Tensor<Float>(1.0)
+    //     return (x + x, y)
+    //   }
+    //
+    // Here foo() returns 2 tensors, but only the first one (as computed by x +
+    // x) is symbolic. The second one for y is concrete, and is computed at
+    // trace creation time, not trace execution time.
+    // Also see the comment block above finalizeAndExecuteTraceFn().
     for (i, output) in outputs.enumerated() {
+      if TFE_TensorHandleIsConcrete(output) != 0 {
```

Comment:
The TF_GraphToFunction call below sets the number of outputs to symbolicOutputs.count. As far as I understand symbolicOutputs.count <= outputs.count after this PR. Should TF_GraphToFunction have outputs.count for the number of outputs?